src/components/cc-orga-member-list/cc-orga-member-list.smart.js

```diff
@@ -0,0 +1,252 @@
+import '../cc-smart-container/cc-smart-container.js';
+import {
+  addMember,
+  getAllMembers,
+  removeMemeber as removeMember,
+  updateMember,
+} from '@clevercloud/client/esm/api/v2/organisation.js';
+import { getId } from '@clevercloud/client/esm/api/v2/user.js';
+import { defineSmartComponent } from '../../lib/define-smart-component.js';
+import { i18n } from '../../lib/i18n.js';
+import { notifyError, notifySuccess } from '../../lib/notifications.js';
+import { sendToApi } from '../../lib/send-to-api.js';
+import { CcOrgaMemberList } from './cc-orga-member-list.js';
+
+const UNAUTHORISED_ADMIN_ADDITION = 6451;
+const UNAUTHORISED_ADMIN_DELETION = 6452;
+
+defineSmartComponent({
+  selector: 'cc-orga-member-list',
+  params: {
+    apiConfig: { type: Object },
+    ownerId: { type: String },
+  },
+  onContextUpdate ({ component, context, onEvent, updateComponent, signal }) {
+
+    const { apiConfig, ownerId } = context;
+
+    /**
+     * Checks if a manager is trying to edit an admin
+     *
+     * @param {OrgaMemberRole} memberToUpdateRole - the current role of the member to update
+     * @return {boolean} - `true` if a manager is trying to edit and admin / `false` otherwise.
+     */
+    function checkIsManagerEditingAdmin (memberToUpdateRole) {
+      const currentUser = component.members.value.find((member) => member.isCurrentUser);
+      const isCurrentUserManager = currentUser.role === 'MANAGER';
+      const isMemberToUpdateAdmin = memberToUpdateRole === 'ADMIN';
+
+      return isCurrentUserManager && isMemberToUpdateAdmin;
+    }
```

Comment:
you can:
- rename the function
- optimize it be failing faster when given role is not 'ADMIN'

```suggestion
    function isManagerEditingAdmin (role) {
      if (role !== 'ADMIN') {
        return false;
      }
      const currentUser = component.members.value.find((member) => member.isCurrentUser);
      return currentUser.role === 'MANAGER';
    }
```